src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -314,10 +316,6 @@ class UpdateInductionTest : IntegrationTestBase() {
       workOnRelease = aValidUpdateWorkOnReleaseRequestForPrisonerLookingToWork(
         reference = persistedInduction.workOnRelease.reference,
       ),
-      previousQualifications = aValidUpdatePreviousQualificationsRequest(
```

Comment:
The qualifications stuff in this test has been removed as the test itself isnt really about qualifications.
I've added a new test at the end that updates an induction with a specific focus on the various changes to qualifications that can be made in the update request 👍